pkg/pillar/build.yml

```diff
@@ -14,7 +14,9 @@ config:
     - /var/config:/config
     - /var/log:/var/log/dom0
     - /:/hostfs
+    - /var/persist:/persist:rshared,rbind
     - /etc/eve-release:/etc/eve-release
+  rootfsPropagation: shared
```

Comment:
why rootfsPropagation here?

Reply:
I thought this and the rootfsPropagation flag below might be required to make changes done by pillar container sync down to host file system and be visible to other containers.